app/models/poll.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class Poll < ActiveRecord::Base
+  has_many :options, dependent: :destroy
+
+  validates :title, presence: true
+
+  after_touch :ensure_status_is_correct
+
+  def make_ready
```

Comment:
What other statuses can a Poll have? Why is this not a `published` boolean flag or (even better) a `published_at` timestamp?